pages/main/listview.php

```diff
@@ -52,6 +52,18 @@
               foreach ($publishedArticles as $article) {
                 if ($index++ < $slides) continue;
                 $published_at = date("F jS, Y", strtotime($article['published_at']));
+
+                $addToFavoritesButton = '';
+                session_start();
+                if ($_SESSION["role"] && strtoupper($_SESSION["role"]) == "USER") {
+                  $articleId = $article['id'];
+                  $addToFavoritesButton = "
+                  <form method='post' action='add_to_favorites.php'>
+                    <input type='hidden' name='articleId' value='$articleId'>
+                    <button type='submit'>Add to Favorites</button>
+                  </form>";
```

Comment:
This seems like it doesnt hava any styling it all, are you sure it looks good enough?